tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/executable/cases/project/NewEspressifIDFProjectTest.java

```diff
@@ -277,6 +289,32 @@ private static void turnOnDfu()
 			bot.comboBox().setSelection("DFU");
 			bot.button("OK").click();
 		}
+		
+		private static void givenESP32LaunchTargetIsSelected()
+		{
+			launchBarTargetSelector.select("esp32");
+		}
+		
+		private static void selectNewDebugLaunchConfig() {
+			givenESP32LaunchTargetIsSelected();
+			launchBarConfigSelector.select("New Launch Configuration...");
+			bot.table(0).select("Debug");
+			bot.table(1).select("ESP-IDF GDB OpenOCD Debugging");
+			bot.button("Next >").click();
+			SWTBotText textWidget = bot.textWithLabel("Project:");
+			assertTrue(textWidget.getText().equals(projectName));
+			bot.cTabItem("Debugger").activate();
+			SWTBotText textWidget1 = bot.textWithLabel("Actual Executable:");
+			String pathtoexe = System.getProperty("user.home");
+			Path p = Paths.get(pathtoexe
+					+ "\\.espressif\\tools\\xtensa-esp-elf-gdb\\12.1_20221002\\xtensa-esp-elf-gdb\\bin\\xtensa-esp32-elf-gdb.exe");
```

Comment:
it's bad to rely on the exact version since different esp-idf and tools can be provided in the property file. I think we should get the actual executable based on the Eclipse environment variable and regular expression.

Reply:
@sigmaaa Thats a good idea but I think for testing we can rely on this but we need to have different configurations based on that. I think we need to sooner or later add some config property for the IDF version under test and based on that fetch correct property value. I am not sure if that will be a good approach or not or this regex will be better. Maybe we should have a call to discuss this properly.